versions.properties

```diff
@@ -8,11 +8,11 @@ version.androidx.activity=1.1.0
 version.androidx.appcompat=1.1.0
 version.androidx.constraintlayout=2.0.1
 version.androidx.databinding=4.0.0
-version.androidx.fragment=1.2.5
+version.androidx.fragment=1.3.0-rc02
 version.androidx.gridlayout=1.0.0
-version.androidx.lifecycle=2.2.0
+version.androidx.lifecycle=2.3.0-rc01
 version.androidx.recyclerview=1.1.0
-version.androidx.savedstate=1.0.0
+version.androidx.savedstate=1.1.0-rc01
 version.androidx.test.core=1.3.0
```

Comment:
The whole ViewTree*Owner set of classes is only available in these pre-release versions. I've discussed using these versions with @rjrjr in the past, internally, and we're fine with it. This being a _library_, I'm not sure it's something we want to do – but we don't really have a choice unless we want to wait to be able to start supporting Compose until all these are released as stable.